catalog/core/catalog-core-metacardtype/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -15,7 +15,7 @@
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
 
     <bean id="commonMetacardType" class="ddf.catalog.data.impl.MetacardTypeImpl">
-        <argument value="common"/>
+        <argument value="fallback.common"/>
```

Comment:
I know we are going with the "fallback" prefix for tika metacards but this is at a "higher" level than just tika. I don't think we should rename this metacard type but create an equivalent type with the fallback prefix for tika.